### PR TITLE
Remove `skey == ""` check from add/edit methods.

### PR DESCRIPTION
**Pull request by sveneberth**
If you make a POST request with valid form data but don't provide a skey, you get no error currently. Only if you provide invalid skey. Let's always raise an 412, no matter if the skey is missing or wrong.

### CODE REVIEW

**Review by ArneGudermann — APPROVED** [review]

File: core/tasks.py [hunk 1]
```diff
@@ -369,8 +369,7 @@ def execute(self, taskID, *args, **kwargs):
             raise errors.Unauthorized()
         skel = task.dataSkel()
         skey = kwargs.get("skey", "")
-        if len(kwargs) == 0 or skey == "" or not skel.fromClient(kwargs) or (
-            "bounce" in kwargs and kwargs["bounce"] == "1"):
+        if len(kwargs) == 0 or not skel.fromClient(kwargs) or kwargs.get("bounce") == "1":
```
Comment (phorward) [inline core/tasks.py:372]: Generally, I would like to propose here to test the "bounce"-parameter on `conf["viur.bone.boolean.str2true"]` but this should be done in a separate PR.

**Review by phorward — APPROVED** [review]
